bin/vcloud-launch

```diff
@@ -2,4 +2,32 @@
 
 require 'vcloud/launcher'
 
-Vcloud::Launcher::Cli.new(ARGV).run
+banner_text = <<EOF
+vcloud-launch takes a configuration describing a vCloud Org,
```

Comment:
Besides, we'd have to argue about whether it's "organiSation" or "organiZation".